packages/prisma/schema.prisma

```diff
@@ -883,6 +883,8 @@ enum WebhookTriggerEvents {
   OOO_CREATED
   AFTER_HOSTS_CAL_VIDEO_NO_SHOW
   AFTER_GUESTS_CAL_VIDEO_NO_SHOW
+  AFTER_HOSTS_GOOGLE_MEET_NO_SHOW
+  AFTER_GUESTS_GOOGLE_MEET_NO_SHOW
```

Comment:
I’m wondering if we really need a separate trigger for this, or if we should handle everything in one webhook. We could name it 'After host didn’t join call' and clarify below at the time input that this is only available for Cal Video and Google Meet

Reply:
Ok makes sense